ffbs-parker-nodeconfig/Makefile

```diff
@@ -21,19 +21,5 @@ define Package/ffbs-parker-nodeconfig/description
   It keeps track of the node's connectivity and changes the state as needed.
 endef
 
-define Build/Prepare
-	mkdir -p $(PKG_BUILD_DIR)
-endef
-
-define Build/Configure
-endef
-
-define Build/Compile
-endef
-
-define Package/ffbs-parker-nodeconfig/install
-	$(CP) ./files/* $(1)/
-endef
-
-$(eval $(call BuildPackage,ffbs-parker-nodeconfig))
+$(eval $(call BuildPackageGluon,ffbs-parker-nodeconfig))
```

Comment:
Is `BuildPackageGluon` preferable over `BuildPackage`? What's the difference?

Reply:
BuildPackageGluon does also include the check_site checks for gluon, sets tags and handles the respondd.c part directly:
https://github.com/freifunk-gluon/gluon/blob/main/package/gluon.mk#L91

Other than that, it is not required, but I would use it for consistency with the other packages as this package relies on gluon anyway